examples/delegator-generation/src/adder.json

```diff
@@ -1,5 +1,5 @@
 {
-  "schema_version": "0.1.0",
+  "schema_version": "0.3.0",
```

Comment:
Let's maybe move this file to some common place and symlink it everywhere it is needed? Kind of awkward to edit it in three different places.